django/db/backends/base/schema.py

```diff
@@ -831,33 +830,28 @@ def _alter_many_to_many(self, model, old_field, new_field, strict):
 
     def _create_index_name(self, model, column_names, suffix=""):
```

Comment:
That could surely be done but such a design change seems more suitable for #6643 (as you have done) than this PR to me.